### PR TITLE
699 categorical.concatenate fails

### PR DESCRIPTION
**Pull request by hokiegeek2**

### CODE REVIEW

File: tests/categorical_test.py [hunk 1]
```diff
@@ -106,4 +106,17 @@ def testBinop(self):
...
         with self.assertRaises(NotImplementedError):
             cat._binop('string 1', '===')
-        
\ No newline at end of file
+            
+    def testConcatenate(self):
+        catOne = self._getCategorical()
+        catTwo = self._getRandomizedCategorical()
+        
+        resultCat = catOne.concatenate([catTwo])
```
Comment (reuster986) [inline tests/categorical_test.py:114]: I'd like to make sure we also test the preferred idiom of `resultCat = ak.concatenate([catOne, catTwo])`, either by changing this line or adding a separate test (even better).

I *think* `ak.concatenate(..., ordered=False)` should also work with Categoricals, but we should explicitly test that one too.

File: arkouda/categorical.py [hunk 2]
```diff
@@ -477,7 +477,7 @@ def concatenate(self, others : List[Categorical], ordered : bool=True) -> Catego
             idxsizes = np.array([self.categories.size] + \
                                 [o.categories.size for o in others])
             idxoffsets = np.cumsum(idxsizes) - idxsizes
-            oldvals = concatenate([c.codes + off for c, off in \
+            oldvals = concatenate([c + off for c, off in \
```
Comment (reuster986) [inline arkouda/categorical.py:480]: nice catch!

**Review by reuster986 — CHANGES_REQUESTED** [review]
Thanks for fixing this! Just one request for an additional test (see inline comment)

File: tests/categorical_test.py (hunk 1 above)
Comment (hokiegeek2) [reply tests/categorical_test.py:114]: @reuster986 Done! Just pushed the commit including the two tests you requested.

**Review by reuster986 — COMMENTED** [review]
Perfect! Thanks!

File: tests/categorical_test.py (hunk 1 above)
Comment (hokiegeek2) [reply tests/categorical_test.py:114]: @reuster986 Please review when you get a chance and lemme know if you have any other changes you'd like to see.

**Review by reuster986 — APPROVED** [review]

**Comment by reuster986** [comment]
Sorry, I thought I had already approved but must not have clicked the right thing!

**Pull request merged**